ssz/utils.py

```diff
@@ -124,7 +124,7 @@ def get_next_power_of_two(value: int) -> int:
     if value <= 0:
         return 1
     else:
-        return 2 ** (value - 1).bit_length()
+        return int(2 ** (value - 1).bit_length())
```

Comment:
`**` will only return non-int if `value` is < 0, which we check for, but mypy doesn't understand that.